From bf5770bb1b25c35a978f446e479e12f9f6805e76 Mon Sep 17 00:00:00 2001
From: Jan Tulak <jtulak@redhat.com>
Date: Tue, 23 Apr 2019 16:15:51 +0200
Subject: [PATCH 9/9] tests: rhel8 changed default luks1 to luks2

Signed-off-by: Jan Tulak <jtulak@redhat.com>
---
 tests/bashtests/012-crypt-create.sh | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/tests/bashtests/012-crypt-create.sh b/tests/bashtests/012-crypt-create.sh
index b668339..bf39637 100755
--- a/tests/bashtests/012-crypt-create.sh
+++ b/tests/bashtests/012-crypt-create.sh
@@ -67,17 +67,17 @@ ssm remove ${DEV}/$crypt_vol1
 
 # Create encrypted volume
 pass | ssm create $dev1
-check crypt_vol_field $crypt_vol1 type LUKS1
+check crypt_vol_field $crypt_vol1 type LUKS2
 check crypt_vol_field $crypt_vol1 device $dev1
 check list_table "$(ssm list vol)" $crypt_vol1 $SSM_CRYPT_DEFAULT_POOL none crypt
 
 pass | ssm create $dev2 -e
-check crypt_vol_field $crypt_vol2 type LUKS1
+check crypt_vol_field $crypt_vol2 type LUKS2
 check crypt_vol_field $crypt_vol2 device $dev2
 check list_table "$(ssm list vol)" $crypt_vol2 $SSM_CRYPT_DEFAULT_POOL none crypt
 
 pass | ssm create -e luks $dev3
-check crypt_vol_field $crypt_vol3 type LUKS1
+check crypt_vol_field $crypt_vol3 type LUKS2
 check crypt_vol_field $crypt_vol3 device $dev3
 check list_table "$(ssm list vol)" $crypt_vol3 $SSM_CRYPT_DEFAULT_POOL none crypt
 
@@ -115,16 +115,16 @@ export SSM_DEFAULT_BACKEND='lvm'
 
 # Try a short password with backend different than crypt
 ! echo -e "a\na" | ssm create $dev1 -e luks
-! check crypt_vol_field $crypt_vol1 type LUKS1
+! check crypt_vol_field $crypt_vol1 type LUKS2
 # force it
 echo -e "a\na" | ssm -f create $dev1 -e luks
-check crypt_vol_field $crypt_vol1 type LUKS1
+check crypt_vol_field $crypt_vol1 type LUKS2
 ssm remove ${DEV}/$crypt_vol1
 ssm -f remove $SSM_LVM_DEFAULT_POOL || true
 
 pass | ssm create --fs $fs3 $dev1 $dev2 $mnt1 -e
 check mountpoint $crypt_vol1 $mnt1
-check crypt_vol_field $crypt_vol1 type LUKS1
+check crypt_vol_field $crypt_vol1 type LUKS2
 check crypt_vol_field $crypt_vol1 device ${SSM_LVM_DEFAULT_POOL}-$lvol1
 check list_table "$(ssm list vol)" $crypt_vol1 $SSM_CRYPT_DEFAULT_POOL none $fs3 none none crypt
 check list_table "$(ssm list vol)" $SSM_LVM_DEFAULT_POOL/$lvol1 $SSM_LVM_DEFAULT_POOL none linear
@@ -138,7 +138,7 @@ check list_table "$(ssm list vol)" $SSM_LVM_DEFAULT_POOL/$lvol2 $SSM_LVM_DEFAULT
 check lv_field $SSM_LVM_DEFAULT_POOL/$lvol2 pv_count 4
 
 pass | ssm create $dev5 -e luks
-check crypt_vol_field $crypt_vol3 type LUKS1
+check crypt_vol_field $crypt_vol3 type LUKS2
 check crypt_vol_field $crypt_vol3 device ${SSM_LVM_DEFAULT_POOL}-$lvol3
 check list_table "$(ssm list vol)" $crypt_vol3 $SSM_CRYPT_DEFAULT_POOL none crypt
 check list_table "$(ssm list vol)" $SSM_LVM_DEFAULT_POOL/$lvol3 $SSM_LVM_DEFAULT_POOL none linear
@@ -161,7 +161,7 @@ ssm  -f remove $SSM_LVM_DEFAULT_POOL
 ssm create $dev1 $dev2
 ssm list
 pass | ssm -b crypt create $DM_DEV_DIR/$SSM_LVM_DEFAULT_POOL/$lvol1
-check crypt_vol_field $crypt_vol1 type LUKS1
+check crypt_vol_field $crypt_vol1 type LUKS2
 check crypt_vol_field $crypt_vol1 device ${SSM_LVM_DEFAULT_POOL}-$lvol1
 check list_table "$(ssm list vol)" $crypt_vol1 $SSM_CRYPT_DEFAULT_POOL none crypt
 check list_table "$(ssm list vol)" $SSM_LVM_DEFAULT_POOL/$lvol1 $SSM_LVM_DEFAULT_POOL none linear
-- 
2.21.0

